.npmignore

```diff
@@ -0,0 +1,12 @@
+.github/
+.idea/
+docs/
+grunt-tasks/
+pdf/
+php/
+sources/
+react-icons/
+svg-min/
+svg-min-react/
+svg-sprite/
+Gruntfile.js
```

Comment:
Is a blacklist the right approach here? Seems like we only want to publish only comparatively few files in the npm, which to me indicates a whitelist (i.e. the `files` attribute in `package.json` is better suited). Curious why you decided against continuing to use that?

Reply:
I've tried whitelisting every individual icon by setting the `files` prop to `*.js` and then blacklist only the `Gruntfile.js` using the `.npmignore`. It seems that in case of conflict the files prop takes precedence over .npmignore, so the Gruntfile.js was still included in the npm package. So I changed it to blacklisting to prevent the Gruntfile.js from being published.

Reply:
(Related thread: https://github.com/Automattic/gridicons/pull/283#discussion_r172838047)